src/javascript/EditPanel/EditPanelHeader/EditPanelHeader.scss

```diff
@@ -0,0 +1,58 @@
+.headerLeft {
+    display: flex;
+    flex-grow: 1;
+    align-items: center;
```

Comment:
The layout must be manage by the moonstone component itself. But if you really need it you can use the utilities classe provided by moonstone `.flexRow` `flexFluid` `alignCenter`

Reply:
replaced with moonstone classes